package.json

```diff
@@ -11,6 +11,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.15.0",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
```

Comment:
- [ ] Good job so far but when we run the app we got en blank page and that is because in your package.json file you specified that the homepage should be `/math-magicians`, please kindly fix that 🙃 

<img width="891" alt="Screenshot 2023-08-22 at 21 45 46" src="https://github.com/AjroudiRida/math-magicians/assets/74501523/fa807eed-39c5-4e89-8a88-f8f3de662e90">